.github/workflows/build-branch.yml

```diff
@@ -1,10 +1,11 @@
-name: Build PR Branch
+name: Build Branch/PR
 
 on:
   push:
     branches-ignore:
       - main
-
+  pull_request:
+      types: [opened, reopened]
```

Comment:
Default seems to be [[opened, synchronize, reopened]](
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request). I think we could also keep the [synchronize](https://github.com/orgs/community/discussions/24567#discussioncomment-3244510) for this project.